site/lib/pure.js

```diff
@@ -10,10 +10,10 @@ const pureFiles = path.resolve(pureDir, 'build');
 const pureMin = fs.readFileSync(path.resolve(pureFiles, 'pure-min.css'), 'utf8');
 
 // use pure-min.css to determine site integrity hash
-const sriHash = crypto.createHash('sha384').update(pureMin, 'utf8').digest('base64');
+const sriHash = crypto.createHash('sha256').update(pureMin, 'utf8').digest('base64');
```

Comment:
Curious why the hash algo needed to change?

Reply:
The jsDelivr shareable links used sha256 so I just changed it to match it, though I guess it wasn't very necessary to change it all aha.

Reply:
I think either one is fine, although the current algo might be more secure, but I'm not sure.

Reply:
A [bit of reading online](https://stackoverflow.com/questions/52101501/what-is-the-best-sri-hash-size) suggests to go with SHA-512, because there’s no reason to use something weaker. Although it does feel a little unnecessary. MDN only mentions sha384 in their docs though and doesn’t mention any other alternative algos.

I think I’ll revert the change back to sha384 since it’s been fine all this time.